Client/Extensions/String+Extension.swift

```diff
@@ -5,11 +5,24 @@
 import Foundation
 
 extension String {
-    /// Handles logic to make part of string bold
+    /// Returns an attributed string in which the first occurrence of the given
+    /// substring is bold.
     /// - Parameters:
-    ///     - boldString: the portion of the string that should be bold. Current string must already include this string.
+    ///     - boldString: the substring that should be bold
     ///     - font: font for entire string, part of string will be converted to bold version of this font
     func attributedText(boldString: String, font: UIFont) -> NSAttributedString {
```

Comment:
`FirefoxAccountSignInViewController` is the only remaining caller of `attributedText(boldString:font:)`. Unlike `SearchViewController`, it doesn't already have an index range handy, so I kept the old signature around instead of changing `FirefoxAccountSignInViewController` to use `attributedText(boldIn:font:)`.